### PR TITLE
Re-enable grab pinning, add kissing.

### PR DESCRIPTION
**Pull request by tigercat2000**
This commit re-enables grab pinning, and adds a new grab extra, kissing.
Kissing can be performed by clicking on someone with the grab in hand, and
intent set to grab.

Pinning is not possible just by upgrading the grab, though.

### CODE REVIEW

**Comment by TheDZD** [comment]
How exactly does one pin a person down now exactly?

**Comment by tigercat2000** [comment]
Oh, you have to have an aggressive grab, and then click on whoever you are grabbing with disarm intent and the grab in hand.

**Comment by Nienhaus** [comment]
ERP

**Comment by TheDZD** [comment]
:-1: on the kissing.
:-1: on the perma pin stun.

(~~No seriously though this is a joke PR, look at the branch name and PR number.~~)

**Comment by Nienhaus** [comment]
ERP IS NOW ALLOWED. Congrats Paradise.

**Comment by Nienhaus** [comment]
Sorry, forced ERP is allowed, not normal ERP.

**Comment by FalseIncarnate** [comment]
Jokes aside... What exactly is the point of kissing other than romantic RP? Guarenteed chance to spread virus infections?

I don't see this being abused by the wrong kind of people at all!

**Comment by tigercat2000** [comment]
@FalseIncarnate That's... actually a brilliant idea, more virus spreading!

**Comment by FalseIncarnate** [comment]
Also, :-1: for not being blocked by masks like CPR is. Practically the same process.

**Comment by Spacemanspark** [comment]
:-1: to the kissing, the fuck.

**Comment by FreeStylaLT** [comment]
I approve of the kissing :love_hotel: :cupid: :see_no_evil:

**Comment by Nomzosha** [comment]
myes :+1:

File: code/modules/mob/mob_grab.dm [hunk 1]
```diff
@@ -317,15 +318,17 @@
...
 						assailant << "<span class='warning'>You no longer pin [affecting] to the ground.</span>"
 						force_down = 0
-						return*///This is a very basic demonstration of a new feature based on attacking someone with the grab, based on intent.
+						return//This is a very basic demonstration of a new feature based on attacking someone with the grab, based on intent.
 								//This specific example would allow you to stop pinning people to the floor without moving away from them.
 					return
 
 				if("grab")
-					return
+					user.visible_message("<span class='warning'>[user] leans in to kiss [affecting]!</span>", "<span class='warning'>You lean in to kiss [affecting].</span>")
```
Comment (Iamgoofball) [inline code/modules/mob/mob_grab.dm]: Are you fucking kidding me right noe

Never ever

**Comment by Spacemanspark** [comment]
If you're really going to add kissing, at least add some confirmation menu for the person who's getting it.
Otherwise you're going to have a bunch of shitlords running around, stunning people, and then forcing them.

**Comment by Robertha89** [comment]
By space law that would be sexual assault, and is a capital crime, and capital crimes carry usually death, i see no reason why not, the pinning, well, if someone is grabbing tightly around your throat, just cause you start resisting a lil doesnt mean automatic get outta jail for free. Honestly I am up for it, as it would properly balance grabs, it would be a way to hold people without cuffs for a bit, and it would be great for RP, both parts, seriously speaking, this is a good thing if we want more rp, punish the abusers instead of the playerbase i say.

**Comment by Iamgoofball** [comment]
We literally have the me verb for shit like kissing
Leave it at that

**Comment by TheDZD** [comment]
# No seriously, if you haven't realized it yet, this is a joke PR.

**Comment by tigercat2000** [comment]
> #1969
Open
tigercat2000 wants to merge 1 commit into ParadiseSS13:master from tigercat2000:1969_OH_YEAH 

>>Add more commits by pushing to the 1969_OH_YEAH branch on tigercat2000/Paradise.

**Comment by Spacemanspark** [comment]
You crossed it out in your first post, so I assumed otherwise, @TheDZD

**Comment by TheDZD** [comment]
It was crossed out for comedic effect, but I suppose that could've been interpreted that way.

**Comment by Fox-McCloud** [comment]
*scream

**Comment by tigercat2000** [comment]
I suppose there is only one thing left to do.
@Markolie requesting review? :P

**Comment by KasparoVy** [comment]
lmao gj

**Comment by mrsdobbins** [comment]
11/10
What could possibly go wrong?

**Comment by Markolie** [comment]
9/11 would merge

**Comment by Earthdivine** [comment]
:+1: Merge nao.

**Comment by Bad-Apple797** [comment]
Newbie doctors and officers using grab instead of pull will now accidentally kiss everyone, goddamn hilarious.

I thought an officer  accidentally starts choking the HOS and doesn't let go it was funny, gunna be twice as good when they accidentally makeout with the clown and get fired for indecent assault

**Comment by Fox-McCloud** [comment]
@Bad-Apple797 

This is a joke PR, it's not gettin' merged =p

**Comment by Fox-McCloud** [comment]
It was fun while it lasted =p

**Comment by tigercat2000** [comment]
kikikikikikiki

**Comment by Earthdivine** [comment]
RIP

**Comment by Nienhaus** [comment]
RIP

**Pull request closed without merging**